strr-platform-web/app/interfaces/platform-api.ts

```diff
@@ -4,7 +4,8 @@ export interface ApiAddress {
   addressLineTwo: string
   city: string
   province: string
-  postalCode: string
+  postalCode: string,
+  locationDescription: string
```

Comment:
Meant to mention this to you. Are we sure we are including the location description? I'm not seeing it in the figma design or the application payload

Reply:
ahh thank you I meant to bring this up to lekshmi. It should be there I think, but I'll double check tomorrow

Reply:
yeah so she'll be adding this in